src/scss/lr7-classic-theme/variables/_list-group.scss

```diff
@@ -0,0 +1,4 @@
+$list-group-active-bg: lighten($brand-primary, 32.5);
+$list-group-active-border: $brand-primary;
+
+$list-group-border: adjust_hue(saturate(darken($list-group-bg, 6), 16), 210);
```

Comment:
$list-group-border is being used to color the background-color of a list-group-item, should be used to color the border of list-group